### PR TITLE
npiperelay: Add version 0.1.0

### LINKED ISSUE #9063: [Request] npiperelay
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** npiperelay

**Description:** npiperelay allows you to access Windows named pipes from WSL

**Homepage:** [github repo](https://github.com/jstarks/npiperelay)

**Download link(s):** https://github.com/jstarks/npiperelay/releases/download/v0.1.0/npiperelay_windows_amd64.zip

**Some indication of popularity/repute:** github stars, forks, mentioned in various X11+Windows forums, mentioned in [1password SSH developer forums](https://1password.community/discussion/128023/ssh-agent-on-windows-subsystem-for-linux)

### PR DESCRIPTION
**Pull request by tecandrew**
Closes #9063

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/npiperelay.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            "url": "https://github.com/jstarks/npiperelay/releases/download/v0.1.0/npiperelay_windows_386.zip",
+            "hash": "600e3f94f0d9f48e68b5a9276849d00d658ae9c18c81541463ed11b1dfe6c998"
+        }
+    },
+    "extract_to": [
+      ""
+    ],
+    "bin": [
+        "npiperelay.exe"
+    ],
```
Comment (issaclin32) [inline bucket/npiperelay.json:21]:
```suggestion
    "bin": "npiperelay.exe",
```
`bin` can be simplified as a string, if there is only 1 item.

File: bucket/npiperelay.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/jstarks/npiperelay/releases/latest/download/npiperelay_windows_amd64.zip"
+            },
+            "32bit": {
+                "url": "https://github.com/jstarks/npiperelay/releases/latest/download/npiperelay_windows_386.zip"
+            }
+        }
```
Comment (issaclin32) [inline bucket/npiperelay.json:33]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/jstarks/npiperelay/releases/download/v$version/npiperelay_windows_amd64.zip"
            },
            "32bit": {
                "url": "https://github.com/jstarks/npiperelay/releases/download/v$version/npiperelay_windows_386.zip"
            }
        }

```
It is better to use the url that contains **version** if possible.
If we use the url for "latest". When the devs update their files, hash error will occur, until the bot catches up and update the manifest. (our bot checks for updates every 2 hours)

**Comment by tecandrew** [comment]
thanks for the review. the recommendations has been applied 😄

**Pull request merged**